post-pip-compile.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -e
+
+# Remove any cruft from a requirements file generated by pip-tools which we don't want to keep
+
+function show_help {
+    echo "Usage: post-pip-compile.sh file ..."
+    echo "Remove any cruft left behind by pip-compile in the given requirements file(s)."
+    echo ""
+    echo "Updates the instructions for re-generating each requirements file, and removes"
+    echo "\"-e\" prefixes which were added to GitHub URLs only so that pip-compile could"
+    echo "process them correctly."
```

Comment:
The help text should probably be updated to reflect what this repo's copy of the script actually does.